fenix/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -1443,6 +1443,23 @@
                 app:destination="@id/translationsDialogFragment"
                 app:popUpTo="@id/translationSettingsFragment"
                 app:popUpToInclusive="true" />
+            <action
+                android:id="@+id/action_translationSettingsFragment_to_automaticTranslationFragment"
```

Comment:
nit: I sense we might want to do some kind of `pref` or `preference` prefixing here to keep everything organized.

Reply:
Done